### PR TITLE
Chore/local backbone for tests

### PR DESCRIPTION
**Pull request by jkoenig134**
This adds a local backbone for testing purposes to the connector tests.

Also fixing some tests

### CODE REVIEW

File: .dev/appsettings.override.json [hunk 1]
```diff
@@ -0,0 +1,108 @@
+{
+    "Cors": {
+        "AccessControlAllowCredentials": true
+    },
+    "Authentication": {
+        "ApiKey": "test"
+    },
```
Comment (tnotheis) [inline .dev/appsettings.override.json:7]: You can remove this

File: .dev/appsettings.override.json [hunk 2]
```diff
@@ -0,0 +1,108 @@
+{
+    "Cors": {
+        "AccessControlAllowCredentials": true
+    },
```
Comment (tnotheis) [inline .dev/appsettings.override.json:4]: I think you can remove this (not sure though)

**Review by tnotheis — CHANGES_REQUESTED** [review]

File: .dev/compose.backbone.yml [hunk 3]
```diff
@@ -56,18 +56,17 @@ services:
 
   seed-database:
     image: postgres
-    volumes:
-      - ./setup-postgres.sql:/app/setup-db/setup-postgres.sql
     environment:
       - PGPASSWORD=Passw0rd
-    command: psql -h postgres -U postgres -d enmeshed -f /app/setup-db/setup-postgres.sql
+      - CONSUMERAPI_VERSION=v4.2.6
```
Comment (tnotheis) [inline .dev/compose.backbone.yml:61]: The consumer api version is duplicated now (line 3 and 61). Can we get rid of that duplication?

Comment (jkoenig134) [reply .dev/compose.backbone.yml:61]: we can, using compose vars.

Comment (tnotheis) [reply .dev/compose.backbone.yml:61]: And do we want to do it? :)

Comment (jkoenig134) [reply .dev/compose.backbone.yml:61]: wo do 🤯🤯🤯🤯🤯🤯🤯🤯🤯🤯🤯🤯🤯🤯

**Review by tnotheis — APPROVED** [review]

File: .dev/scripts/prepareDevEnvironment.sh [hunk 4]
```diff
@@ -8,7 +8,7 @@ if ! type "docker" > /dev/null; then
 fi
 
 if ! type "tsc" > /dev/null; then
-  npm i -g typescript
+  npm i -g typescript cpx
```
[the remark this answers is not in the document]
Comment (jkoenig134) [reply .dev/scripts/prepareDevEnvironment.sh:11]: this is incorrect! we develop on unix. Developers on Windows are required to use wsl, mac users can use this script (but not socat part)

[the remark this answers is not in the document]
Comment (jkoenig134) [reply .dev/scripts/prepareDevEnvironment.sh:17]: additionally this is only a convenience script, everything done here is already stated in the README_dev.md file

File: .dev/scripts/prepareDevEnvironment.sh [hunk 5]
```diff
@@ -1,14 +1,20 @@
...
-if ! type "docker" > /dev/null; then
+if ! type "docker" >/dev/null; then
   echo "docker not found"
   exit 1
 fi
 
-if ! type "tsc" > /dev/null; then
-  npm i -g typescript
+if ! type "tsc" >/dev/null; then
+  npm i -g typescript cpx
```
Comment (tnotheis) [inline .dev/scripts/prepareDevEnvironment.sh:17]: Just realized that now you only install cpx if typescript is not installed. :)

**Review by tnotheis — APPROVED** [review]

**Review by tnotheis — APPROVED** [review]

**Pull request merged**